tools/pdp-metadata/pdp-metadata.js

```diff
@@ -144,6 +145,7 @@ const getProducts = async (config, pageNumber) => {
           meta_description: finalDescription,
           'og:image': baseImageUrl,
           'og:image:secure_url': baseImageUrl,
+          'last-modified': lastModifiedAt,
```

Comment:
is this in `ISO` format?

Reply:
@rofe It is. You can double check the resulting file here: https://www.aemshop.net/products/default/metadata.json.